.github/workflows/copy_labels.yml

```
@@ -6,6 +6,9 @@ on:
 
 jobs:
   copy-labels:
+    if: "! github.event.pull_request.head.repo.fork && github.actor != 'dependabot[bot]'"
```

```suggestion
  copy-labels:
    # Avoid being triggered by forks
    if: "! github.event.pull_request.head.repo.fork && github.actor != 'dependabot[bot]'"
```

I suppose this is better than `if: ${{ github.repository_owner == 'IQSS' }}` because it will still work if we move the repo to the gdcc org or something. "IQSS" isn't hard coded.

---

Also, I believe that your proposed change (copied from other workflows in this repo) doesn't work here. The pull_request runs in the context of the source repo. AFAIK the "github.repository_owner" is always set to the target repo. I might be wrong though.